### PR TITLE
Add Simple Chinese Translation and Picture

### PR DESCRIPTION
**Pull request by fwx5618177**
- Add Simpled Chinese Translation.

### CODE REVIEW

**Comment by fwx5618177** [comment]
PTAL. @cpettitt @mwaldstein @hypergeometric @styfle

**Comment by fwx5618177** [comment]
Here is feature issue: https://github.com/dagrejs/graphlib/issues/183

File: README.md [hunk 1]
```diff
@@ -12,8 +13,8 @@ There are 2 versions on NPM, but only [the one in the DagreJs org](https://www.n
 
 # License
 
-Graphlib is licensed under the terms of the MIT License. See the
```
Comment (rustedgrail) [inline README.md:15]: Was this supposed to be removed? The license should still be included in the readme.

File: README_ZH.md
Comment (rustedgrail) [inline README_ZH.md:1]: Should this be renamed _CN.md? The capitalization should match between the filename and the link in English readme, right?

**Review by rustedgrail — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (fwx5618177) [reply README.md:15]: > 

I think you're right.

File: README_CN.md [hunk 2]
```diff
@@ -0,0 +1,515 @@
+# 进度
+- [x] 翻译 [API](https://github.com/dagrejs/graphlib/wiki/API-Reference) - 2022/03/23 21:06
+
+> 中文 | [English](ReadMe.md)
```
Comment (rustedgrail) [inline README_CN.md:4]: Also the casing needs to match here, README.md

**Review by rustedgrail — APPROVED** [review]

**Pull request merged**